pkg/registry/rbac/helpers.go

```diff
@@ -44,6 +44,7 @@ func IsOnlyMutatingGCFields(obj, old runtime.Object, equalities conversion.Equal
 	copiedMeta.SetOwnerReferences(oldMeta.GetOwnerReferences())
 	copiedMeta.SetFinalizers(oldMeta.GetFinalizers())
 	copiedMeta.SetSelfLink(oldMeta.GetSelfLink())
+	copiedMeta.SetManagedFields(oldMeta.GetManagedFields())
```

Comment:
Seems like we want to deny taking ownership of fields outside of the GC fields, right?

Reply:
yes, but it's fairly hard to inspect the managedFields to make this kind of validation.

Reply:
Keeping this check coarse-grained at the managedFields level seems ok to me, but TestIsOnlyMutatingGCFields needs updating to prove this is effective.

Reply:
oops, missed there was already a test